### ISSUE TITLE
Reverse slider direction

### ISSUE THREAD

**Issue by joelworsham** [andreruffert/rangeslider.js#286]
Is it possible? For me specifically, I want to reverse the vertical direction so the top of the slider is the smallest value and the bottom is the highest value.

**Comment by PardeepWildnet** [comment]
Same in case of HORIZONTAL direction , is it possible to reverse the order from right to left ?

**Comment by joelworsham** [comment]
I found a workaround. Set a hidden input as the input whose value is used, and then simply update that input value onInit and onSlide like so:

```
onSlide: function (position, value) {

  let $altInput = $('#alt-input');
  let newValue       = this.max - this.value + this.min;

  $altInput.val(newValue);
}
```

**Comment by NayanKhedkar** [comment]
@joelworsham  yes,it is possible.This feature has been implemented in PR:https://github.com/andreruffert/rangeslider.js/pull/245
